saltstack/base/salt/ccm-client/init.sls

```diff
@@ -0,0 +1,19 @@
+/cdp/bin/reverse-tunnel.sh:
+  file.managed:
+    - name: /cdp/bin/reverse-tunnel.sh
+    - source: salt://{{ slspath }}/cdp/bin/reverse-tunnel.sh
+
+/cdp/bin/reverse-tunnel-values.sh:
+  file.managed:
+    - name: /cdp/bin/reverse-tunnel-values.sh
+    - source: salt://{{ slspath }}/cdp/bin/reverse-tunnel-values.sh
+
+/cdp/bin/update-reverse-tunnel-values.sh:
+  file.managed:
+    - name: /cdp/bin/update-reverse-tunnel-values.sh
+    - source: salt://{{ slspath }}/cdp/bin/updatereverse-tunnel-values.sh
+
+/systemd/tunnel@.service:
```

Comment:
We should enable the service IMHO here: https://docs.saltstack.com/en/latest/ref/states/all/salt.states.service.html#salt.states.service.running